src/main/java/io/sniffy/util/Range.java

```diff
@@ -0,0 +1,62 @@
+package io.sniffy.util;
+
+import io.sniffy.Count;
+import io.sniffy.Expectation;
+
+public class Range {
+
+    public final int value;
+    public final int min;
+    public final int max;
+
+    public Range(int value, int min, int max) {
+        this.value = value;
+        this.min = min;
+        this.max = max;
+    }
+
+    public static Range parse(Expectation expectation) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method parse() has an NPath complexity of 432](https://www.codacy.com/app/sniffy/sniffy/file/2495635952/issues/source?bid=3005960&fileBranchId=3034483#l18)